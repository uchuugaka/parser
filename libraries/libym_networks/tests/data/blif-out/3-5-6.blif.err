"3-5-6.blif" line: 6, column: 12 [ER_UNDEF01]: c: Undefined.
Error in reading 3-5-6.blif
