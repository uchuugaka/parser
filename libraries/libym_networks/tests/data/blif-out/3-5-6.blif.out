.model test1
	["3-5-6.blif" line: 3, columns:1-6]
	["3-5-6.blif" line: 3, columns:8-12]
  inputs elem: a
	["3-5-6.blif" line: 4, column: 9]
  inputs elem: b
	["3-5-6.blif" line: 4, column: 11]
  outputs elem: x
	["3-5-6.blif" line: 5, column: 10]
.names
a
	["3-5-6.blif" line: 6, column: 8]
b
	["3-5-6.blif" line: 6, column: 10]
c
	["3-5-6.blif" line: 6, column: 12]
x
	["3-5-6.blif" line: 6, column: 14]
111 1
.end
	["3-5-6.blif" line: 8, columns:1-4]
error_exit
